hybrid/compartment_examples/inter_comp_call/secure/shared.S

```diff
@@ -53,14 +58,11 @@ switch_compartment:
     ldr       x11, [x29, x11]
     mov       sp, x11
 
-    // Derive a new clr to restore PCC, and store it.
-    cvtp      c11, lr
-
     // Install compartment DDC
     msr       DDC, c1
 
     // Save old DDC (c2), old SP (x12), old LR (c11) on stack
```

Comment:
The comment (`c11`) is now out of date.

Reply:
Fixed in https://github.com/capablevms/cheri-examples/pull/56/commits/8ca60bd06adbbe2c55ab81f33a6cd537b175b4c5.